src/add-to-app/android/project-setup.md

```diff
@@ -137,9 +157,10 @@ module an embeddable Android library.
   To avoid Dex merging issues, `flutter.androidPackage` should not be identical to your host app's package name
 {{site.alert.end}}
 
-### Java 11 requirement
+#### Java 11 requirement
+{:.no_toc}
 
-The Flutter Android engine uses Java 11 features.
+The Flutter Android Gradle plugin uses Java 11 features.
```

Comment:
Do you have more context on this change?

Are the specifics important anyway? Could we just say "Flutter requires Java 11 compatibility."?

Reply:
If this was about downloading Java, I think it wouldn't be necessary, but this is about a configuration option during Android app setup. Many Android apps that would be investigating add-to-app may still have these set to 1.8 (Java 8) since Android was a bit behind the times for a while.

Reply:
According to [our source code](https://github.com/flutter/flutter/blob/main/packages/flutter_tools/test/general.shard/android/gradle_errors_test.dart#L1101-L1116), the Gradle plugin requires Java 11 exactly.

Reply:
https://github.com/flutter/flutter/blob/72b69f944915e8aae2af6a5475d4d6d4f4a4e4af/packages/flutter_tools/lib/src/android/gradle_errors.dart#L631

I think this is the code being tested and in that we are saying java 11 or higher.